web/src/main/java/co/mobileaction/example/web/service/IPostQueueService.java

```diff
@@ -8,4 +8,5 @@
 public interface IPostQueueService
 {
     void sendPostRequestForAllItems();
+    int enqueueDistinctUserIdsForCrawl();

```

Comment:
Leave a space between methods
```suggestion

    int enqueueDistinctUserIdsForCrawl();
```